ui/app/AppLayouts/Wallet/popups/AddAccountModal.qml

```diff
@@ -140,9 +141,10 @@ StatusModal {
                     id: passwordInput
                     anchors.fill: parent
 
-                    placeholderText: qsTr("Enter your password…")
+                    placeholderText: qsTr("Enter your password...")
                     label: qsTr("Password")
                     textField.echoMode: TextInput.Password
+                    textField.objectName: "password"
```

Comment:
This objectName might need to be more descriptive, so we don't clash or confuse with the login password and such

Reply:
do u have a suggestion? maybe accountModalPassword
even if I doubt we will display login view with add account modal

Reply:
Yeah that sounds better!